### ISSUE TITLE
Theme radio buttons does not behave as expected

### ISSUE THREAD

**Issue by eddyfadeev** [lencx/Noi#202]
I have a dark system theme set in Windows, and I've configured the theme radio button to match the system settings. However, when I try to force the dark theme, it doesn't change immediately. Instead, I have to switch to the light theme first and then back to the dark theme to make it work. The same issue occurs when switching from dark to system. I didn't try with a light theme, but I think it will behave in the way. You can check the recording in the attachments for more details.

https://github.com/user-attachments/assets/fb4df00d-3977-40f6-9f34-36eff8cc9e0f

**Comment by lencx** [comment]
You can try clicking the small circle in front of the text.
<img width="398" alt="Screenshot 2024-07-21 at 22 34 21" src="https://github.com/user-attachments/assets/6f274f7e-56cf-4269-ae3d-7c0c22ec0931">

**Comment by eddyfadeev** [comment]
Yes, I did. It was only for the video, so you can see an actual click
